Xamarin.Forms.Platform.UAP/DatePickerRenderer.cs

```diff
@@ -142,6 +142,83 @@ void UpdateDate(DateTime date)
 		{
 			if (Control != null)
 				Control.Date = new DateTimeOffset(new DateTime(date.Ticks, DateTimeKind.Unspecified));
+
+			UpdateDay();
+			UpdateMonth();
+			UpdateYear();
+		}
+
+		void UpdateMonth()
+		{
+			Control.MonthVisible = true;
+			if (String.IsNullOrWhiteSpace(Element.Format) || Element.Format.Equals("d"))
```

Comment:
Move this check to a helper or a method? It looks like this same check is performed a few times.